src/main/java/io/debezium/connector/vitess/VitessSourceInfoStructMaker.java

```diff
@@ -35,9 +36,10 @@ public Schema schema() {
     @Override
     public Struct struct(SourceInfo sourceInfo) {
         final Struct res = super.commonStruct(sourceInfo)
+                .put(SourceInfo.KEYSPACE_NAME_KEY, sourceInfo.getTableId().schema())
```

Comment:
I don't know about JDBC, but it doesn't seem really matter to us since we use it internally to pass through keyspace. If we figure out that catalog is better then we can refactor it.